### PR TITLE
Commerce 4836 added new field for datasetdislpay

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/rest-openapi-v2.0.yaml [hunk 1]
```diff
@@ -612,6 +612,9 @@ components:
                 hasTierPrice:
                     example: true
                     type: boolean
+                hasTierPriceString:
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/rest-openapi-v2.0.yaml:615]: @riccardo-alberti  fyi

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
This is not the right fix

**Pull request closed without merging**